src/components/pages/HomePage/Countries.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Country from './Country';
+import { getCountries } from '../../../redux/CountriesState/countriesState';
+
+const Countries = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getCountries());
+  }, []);
+  const countries = useSelector((state) => (state.countries));
+  return (
+    <div className="countriesPage" data-testid="countriesContent">
+      <div className="homePageCover">
+        <h1>World Weather</h1>
+        <p>Choose the country</p>
+      </div>
+      <div className="padding">
+        <h2 className="countriesHeader">Countries</h2>
+      </div>
+      <div className="padding countries">
+        {Object.values(countries).map((country) => (
+          <Country
+            key={country.iso3}
+            name={country.name}
+            states={country.states}
+            iso3={country.iso3}
+          />
+        ))}
+      </div>
```

Comment:
- [ ] Kindly stick to the original design as much as possible. Please make sure that the list of countries is displayed with 2 countries in each row, to match the original design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/177815366-58c1bc4e-c64f-479f-ba99-df83ad039e96.png) | ![image](https://user-images.githubusercontent.com/62290545/177815716-5f4e3dfb-226d-4dcf-a51e-79e48d1e8db8.png) |

Reply:
Well resolved 👍🏽